File: 7-kyu/make-the-small-words-big!/javascript/solution.js
const vowelHelper = letter => !"aeiou".includes(letter.toLowerCase());

const smallWordHelper = sentence => {
  return sentence.split(" ")
    .map((word) => word.length < 4 ? word.toUpperCase() : word.split("").filter(vowelHelper).join(""))
    .join(" ");
};

File: 7-kyu/make-the-small-words-big!/python/solution.py
def remove_vowels(word):
    return ''.join(ch if ch.lower() not in 'aeiou' else '' for ch in word)

def small_word_helper(sentence):
    return ' '.join(word.upper() if len(word) < 4 else remove_vowels(word) for word in sentence.split(' '))